src/components/drop-down/drop-down-choice.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+interface DropDownChoiceProps {
+  variant?: 'dark' | 'light';
+  className?: string;
+  bubbleOnClickEvent?: boolean;
+  onClick: (event: React.MouseEvent) => void;
+}
+
+export const DropDownChoice = ({
```

Comment:
Is this used anywhere?

Reply:
I forgot to remove it, haha. I'll fix that right now